sdk/metricsadvisor/perf-tests/ai-metrics-advisor/test/metricsAdvisor.spec.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressTest, getEnvVar } from "@azure/test-utils-perfstress";
+import { MetricsAdvisorClient, MetricsAdvisorKeyCredential } from "../../../ai-metrics-advisor/src";
```

Comment:
Import from package instead of sources:

```suggestion
import { MetricsAdvisorClient, MetricsAdvisorKeyCredential } from "@azure/ai-metrics-advisor";
```

Example: 
https://github.com/Azure/azure-sdk-for-js/blob/91496d9/sdk/storage/perf-tests/storage-blob/test/storageTest.spec.ts#L6-L10